index.html

```diff
@@ -14,6 +15,79 @@
 </head>
 
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+  <div class="container">
+    <header>
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+      <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
```

Comment:
While this is fine, consider separating out your tags a bit more (so that you don't have two on the same line) to make it slightly more readable:
```html
<a href="index.html">
  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
</a>
```
or, when you have a lot of attributes on a tag, you could even go as far as doing this for readability:
```html
<a href="index.html">
  <img 
    class="logo"
    src="img/logo.png"
    alt="Great Idea! Company logo."
  >
</a>
```